bin/lint-staged

```diff
@@ -34,6 +34,7 @@ cmdline
   .option('-x, --shell', 'Skip parsing of tasks for better shell support')
   .option('-q, --quiet', 'Disable lint-staged’s own console output')
   .option('-d, --debug', 'Enable debug mode')
+  .option('-c, --concurrent', 'Run lint-staged in concurrent mode')
```

Comment:
I think it’s redundant to mention the program’s name here; maybe ”_run tasks in patallel_” like `Listr` describes it. Also, I think this could be a number specifying the actual concurrency. Do you think that would be too complex an option?

Reply:
Also apply the description change to the JSDoc in `lintStaged` and `runAll`.